--- google_appengine/google/appengine/tools/devappserver2/php/setup.php	2015-02-11 20:09:52.000000000 -0800
+++ google_appengine/google/appengine/tools/devappserver2/php/setup-new.php	2015-02-25 01:55:45.371939847 -0800
@@ -80,9 +80,6 @@
         new \google\appengine\runtime\RemoteApiProxy(
           getenv('REMOTE_API_HOST'), getenv('REMOTE_API_PORT'),
           getenv('REMOTE_REQUEST_ID')));
-      $unsetEnv('REMOTE_API_HOST');
-      $unsetEnv('REMOTE_API_PORT');
-      $unsetEnv('REMOTE_REQUEST_ID');
     }
   };
 
